programs/svm-spoke/src/event.rs

```diff
@@ -23,6 +23,13 @@ pub struct EnabledDepositRoute {
     pub enabled: bool,
 }
 
+#[event]
+pub struct RelayRootBundleEvent {
```

Comment:
this is named `RelayedRootBundle` in EVM. Also the ordering is different (rootBundleId, relayerRefundRoot, slowRelayRoot)